src/BinSkim.Sdk/BinaryAnalyzerContext.cs

```diff
@@ -34,12 +33,6 @@ public IBinary Binary
             set => this.iBinary = value;
         }
 
-        public override Exception TargetLoadException
```

Comment:
>TargetLoadException [](http://example.com/codeflow?start=34&length=19)

We now have runtime exceptions hanging off the context object. I think this change breaks target load exceptions! But no unit tests fail. I will go confirm this point.